src/TraceEvent/EventPipe/NetTraceFormat.md

```diff
@@ -201,8 +201,9 @@ Compressed events are encoded with a header:
   - otherwise use the previous MetadataId
 - SequenceNumber optional varuint32 
  - if Flags & 2 the value is read from the stream + previous SequenceNumber
- - otherwise previous sequence number
- - in either case increment SequenceNumber by 1
+ - otherwise if MetaDataId != 0 increment previous sequence number by 1, else keep previous sequence number
```

Comment:
This spec is for the NetTrace V6 format which doesn't encode metadata using events at all (metadataId is always non-zero in V6). The spec you need to update is https://github.com/microsoft/perfview/blob/main/src/TraceEvent/EventPipe/NetTraceFormat_v5.md